src/main/resources/l10n/JabRef_da.properties

```diff
@@ -1760,3 +1760,4 @@ One_file_found=
 The_import_finished_with_warnings\:=
 There_was_one_file_that_could_not_be_imported.=
 There_were_%0_files_which_could_not_be_imported.=
+Global_search=
```

Comment:
I don't know where it exactly is used, but we already have a resource for `Search_globally=Search_globally`